src/jbi/whiteboard_actions/default.py

```diff
@@ -31,10 +37,111 @@ def __init__(self, **kwargs):
         """
-        print(payload)
-        print(self.parameters)
+        try:
+            payload: BugzillaWebhookRequest = BugzillaWebhookRequest.parse_obj(
+                payload
+            )  # typing assistance
+            bugzilla_client = get_bugzilla()
+            jira_client = get_jira()
+            settings = get_settings()
+
+            current_bug_info = bugzilla_client.getbug(payload.bug.id)
+            bug_obj = BugzillaBug.parse_obj(current_bug_info.__dict__)
+            target = payload.event.target
+            linked_issue_key = bug_obj.extract_from_see_also()
+
+            if target == "comment":
+                if not linked_issue_key:
+                    # noop
+                    return JSONResponse(content={"status": "noop"}, status_code=201)
+                # else
+                jira_response = jira_client.issue_add_comment(
+                    issue_key=linked_issue_key,
+                    comment=payload.map_as_jira_comment(),
+                )
+                return JSONResponse(
+                    content={"status": "comment", "jira_response": jira_response},
+                    status_code=201,
+                )
+
+            if target == "bug":
+                if linked_issue_key:
+                    # update
+                    fields, comments = payload.map_as_tuple_of_field_dict_and_comments()
+                    jira_update_response = jira_client.update_issue_field(
+                        key=linked_issue_key, fields=fields
+                    )
+                    # comment
+                    jira_comment_responses = []
+                    for comment in comments:
+                        jira_comment_responses.append(
+                            jira_client.issue_add_comment(
+                                issue_key=linked_issue_key, comment=comment
+                            )
+                        )
+                    return JSONResponse(
+                        content={
+                            "status": "update",
+                            "jira_update_response": jira_update_response,
+                            "jira_comment_responses": jira_comment_responses,
+                        },
+                        status_code=201,
+                    )
+                # else: create jira issue
+                fields = bug_obj.get_jira_issue_dict()
+                fields["project"] = {"key": self.jira_project_key}
+
+                response = jira_client.create_issue(fields=fields)
+
+                if isinstance(response, list):
+                    response = response[0]
+
+                if isinstance(response, dict):
+                    if any(
+                        element in ["errors", "errorMessages"] and response[element]
+                        for element in response.keys()
+                    ):
+                        # Failure to create: err keys exist with value
+                        return None
```

Comment:
Will this return some 500 error from our server? Should we handle this and surface whatever error messages Jira send back?

Reply:
Currently, the behavior is: let the message go, have user re-trigger through bump/minor-update on card. If issues still occur seek manual intervention.

There's some additional handling for retries _we might_ want to consider adding here post-launch; I think we can continue the current process. 

_(Historical: Initially, in v1, there were an attempts at retries; this resulted in some inconsistencies with duplicate creations and duplicate jira issues--with and without tags. There's gonna be some finesse required for this.)_